hack/devtools/local_dev_env.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+# Local development environment script.
+# Execute this script from the root folder of the repo (ARO-RP).
+# This script is aimed to provide an automatic and easy way to prepare 
+# the environment and execute the ARO RP locally.
+# The steps here are the ones defined in docs/deploy-development-rp.md
+# We recommend to use this script after you understand the steps of the process, not before.
+
+
+build_development_az_aro_extension() {
+    echo "INFO: Building development az aro extension..."
+    make az
+}
+
+verify_aro_extension() {
+    echo "INFO: Verifying aro extension..."
+    grep -q 'dev_sources' ~/.azure/config || cat >>~/.azure/config <<EOF
+[extension]
+dev_sources = $PWD/python
+EOF
+}
+
+set_storage_account() {
+    echo "INFO: Setting storage account..."
+    export SECRET_SA_ACCOUNT_NAME=rharosecretsdev make secrets
+}
+
+ask_to_create_default_env_config() {
+    local answer
+    read -p "Do you want to crate a default env file? (existing one will be overwritten, if any) (y / n) " answer
+
+    if [[ "$answer" == "y" ]]; then
+        create_env_file
+    elif [[ "$answer" == "n" ]]; then
+        echo "INFO: Skipping creation of default env file..."
+    else
+        echo "INFO: Unknown option, skipping step..."
+    fi
+}
+
+create_env_file() {
+    echo "INFO: Creating default env config file..."
+
+    cat >env <<EOF
+export LOCATION=eastus
```

Comment:
We have a dev environment in westeurope as well - should we allow that as an option?

Reply:
As this is an optional step where the user can use her/his own env file, I would leave it as it is for simplicity (with the same values as explained [here](https://github.com/Azure/ARO-RP/blob/master/docs/deploy-development-rp.md)).